frontends/web/src/components/keystoreconnectprompt.tsx

```diff
@@ -18,40 +18,68 @@ import { useTranslation } from 'react-i18next';
 import { Button } from './forms';
 import { cancelConnectKeystore, syncConnectKeystore } from '../api/backend';
 import { useSubscribeReset } from '../hooks/api';
+import { Dialog, DialogButtons } from './dialog/dialog';
+import { BitBox02StylizedDark, BitBox02StylizedLight, Cancel, PointToBitBox02 } from './icon';
+import styles from './keystoreconnectprompt.module.css';
+import { useDarkmode } from '../hooks/darkmode';
 import { SkipForTesting } from '../routes/device/components/skipfortesting';
 
+
 export function KeystoreConnectPrompt() {
   const { t } = useTranslation();
+  const { isDarkMode } = useDarkmode();
 
   const [data, reset] = useSubscribeReset(syncConnectKeystore());
   if (!data) {
     return null;
   }
   switch (data.typ) {
   case 'connect':
-    // TODO: make this look nice.
     return (
-      <>
-        { data.keystoreName === '' ?
+      <Dialog title="Connect BitBox02" medium open>
+        <p className={styles.text}>{ data.keystoreName === '' ?
           t('connectKeystore.promptNoName') :
           t('connectKeystore.promptWithName', { name: data.keystoreName })
-        }
-        {/* Software keystore is unlocked from the app, so we add the button here.
```

Comment:
this comment line is missing below

Reply:
SkipForTesting should be conditionally rendered see:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/b3f968702a030a55eca5eadb1582de1ae8848b5a/frontends/web/src/routes/device/waiting.tsx#L32

it's a bit ugly, maybe you could refactor that SkipForTesting does this logic internally?